package.json

```diff
@@ -24,7 +24,8 @@
     "version": "lerna version",
     "publish": "lerna publish from-package",
     "local:publish": "lerna exec -- npm publish --registry http://localhost:4873",
-    "local:unpublish": "lerna exec -- npm unpublish -f --registry http://localhost:4873"
+    "local:unpublish": "lerna exec -- npm unpublish -f --registry http://localhost:4873",
+    "watch": "node watcher.js"
```

Comment:
can we replace the `dev` task with this if it works better?

Reply:
Yeah yeah, currently I want to test for a couple of days. So kept as a side script